### PR TITLE
Clob and Blob implementations

### PR DESCRIPTION
**Pull request by fedefernandez**
This PR provides the implementation for Clob and Blob

Please @juanpedromoreno could you review? Thanks

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -36,4 +36,6 @@ resolvers ++= Seq(
 )
 
 libraryDependencies ++= Seq(
+  "org.specs2" %% "specs2-core" % "3.3.1" % "test",
+  "org.specs2" % "specs2-mock_2.11" % "3.3.1" % "test",
```
Comment (juanpedromoreno) [inline build.sbt]: Maybe we can use `%%` instead of `_2.11`

File: src/main/java/org/sqldroid/SQLDroidConnection.java [hunk 2]
```diff
@@ -56,8 +58,8 @@
      * @param info Properties object with options.  Supported options are "timeout", "retry", and "shared".
      */
     public SQLDroidConnection(String url, Properties info) throws SQLException {
-        Log.v("SQLDroidConnection: " + Thread.currentThread().getId() + " \"" + Thread.currentThread().getName() + "\" " + this);
-        Log.v("New sqlite jdbc from url '" + url + "', " + "'" + info + "'");
+        Log.v("SQLDroid", "SQLDroidConnection: " + Thread.currentThread().getId() + " \"" + Thread.currentThread().getName() + "\" " + this);
```
Comment (juanpedromoreno) [inline src/main/java/org/sqldroid/SQLDroidConnection.java:61]: We are repeating "SQLDroid" in several parts of the code. Pull up to a val?

Comment (fedefernandez) [inline src/main/java/org/sqldroid/SQLDroidConnection.java:61]: Yes, that's true but this changes are in Java code in order to make it compiles. We can fix it when move the code to Scala. Makes sense?

File: src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+import java.io.{ByteArrayInputStream, InputStream, OutputStream}
+import java.sql.{SQLException, Blob, SQLFeatureNotSupportedException}
+
+class SQLDroidBlob(byteArray: Array[Byte]) extends Blob {
+
+  private[this] val arrayLength = Option(byteArray) map (_.length) getOrElse 0
+
+  def getBytes(pos: Long, length: Int): Array[Byte] = {
+    val newPos = pos - 1
```
Comment (juanpedromoreno) [inline src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala:11]: And what does it happen if I specify 0? Does it make sense as default value parameters?

File: src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    throw new SQLFeatureNotSupportedException
+
+  override def toString: String = {
+    Option(this.byteArray) match {
+      case Some(b) =>
+        val prefix = "Blob length %d".format(b.length)
+        val toPrint = if (b.length > 10) b.slice(0, 10) else b
+        val hexString = toPrint map ("0x" + Integer.toHexString(_)) mkString " "
+        val charString = toPrint map (b => "(" + Character.toString(b.toChar)) mkString " "
+        prefix + " " + hexString + " " + charString
```
Comment (juanpedromoreno) [inline src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala]: Use string Interpolator

File: src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala (hunk 3 above)
Comment (juanpedromoreno) [inline src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala:11]: IMO is not developer friendly, we usually use 0 as initial position, makes sense? Maybe I'm missing something

Comment (fedefernandez) [inline src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala:11]: I don't like it, but is that the documentation says :cry: [Blob.getBytes](http://docs.oracle.com/javase/7/docs/api/java/sql/Blob.html#getBytes%28long,%20int%29)

I think that I miss the override prefix, sorry

On the other hand, a default value is allowed when override Java parameters?

File: build.sbt (hunk 1 above)
Comment (fedefernandez) [inline build.sbt]: Good point

File: src/main/java/org/sqldroid/SQLDroidConnection.java (hunk 2 above)
Comment (juanpedromoreno) [inline src/main/java/org/sqldroid/SQLDroidConnection.java:61]: Yes.

File: src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala (hunk 3 above)
Comment (juanpedromoreno) [inline src/main/scala/com/fortysevendeg/android/sqlite/SQLDroidBlob.scala:11]: :+1:

**Comment by juanpedromoreno** [comment]
Added minor comments, once addressed or explained, LGTM ;)

Good work and good test coverage.

**Comment by fedefernandez** [comment]
Thanks for your help @juanpedromoreno

**Pull request merged**